src/blocks.js

```diff
@@ -2814,10 +2814,17 @@ BlockMorph.prototype.userMenu = function () {
                     );
                 }
             } else if (this.selector !== 'evaluateCustomBlock') {
-                menu.addItem(
-                    "hide",
-                    'hidePrimitive'
-                );
+                if (this.isHidden()) {
+                    menu.addItem(
+                        'show',
```

Comment:
This text is a little bit vague (given that you can see the block), but I think it makes enough sense in the context, and given that it's the opposite of hide, I think it's fine.

Reply:
Would un-hide be clearer?